docker/Dockerfile

```diff
@@ -33,6 +33,7 @@ ENV PGRST_DB_URI= \
     PGRST_SERVER_PROXY_URI= \
     PGRST_JWT_SECRET= \
     PGRST_SECRET_IS_BASE64=false \
+    PGRST_JWT_AUD=\
```

Comment:
Actually since I made my other comment about the changelog adjustment let me throw this in too: can you add a space between the equals sign and the backslash?

Sorry to be annoying, the whitespace probably doesn't matter, I just felt kind of OCD about it. 😁